docs/concepts/gas-enabled-network.md

```diff
@@ -0,0 +1,102 @@
+However, the inner private transaction isn't available to non-participants,
+therefore members who aren't party to the transaction can't calculate the intrinsic gas.
+Therefore, the private transaction doesn't consume any gas.
+
+### Raft
+
+The transaction cost is rewarded to the etherbase account (aka coinbase account), which is the default primary local account.
+The etherbase account can be set on startup using the command line flag `--miner.etherbase`.
+Note that for technical reasons the etherbase account can't be changed at runtime using `miner.SetEtherbase()`.
+
+If `--mine` is not provided on the command line then the (leader) node still mints blocks,
+but `--miner.gasprice` is ignored.
+Also, the txpool default `gasPrice` won’t be set to `1000000000`.
+Therefore, we suggest starting all raft nodes with `--mine`.
+
+Standard geth behaviour is that if a transaction has less gas than the accepted minimum
+(set using `--miner.gasprice`) then it is not mined by remote miner nodes.
+On Raft, there is no rotation of miners, so if the local node is not the minter then the transaction remains pending
+until it is resubmitted with an above minimum gas value (or until the local node becomes the miner).
+
+Raft inherits the static block reward that is awarded to miners on upstream geth.
+The static block reward amount varies according to the current release, with the following values currently defined:
+
+- `FrontierBlockReward`           = big.NewInt(5e+18) // Block reward in wei for successfully mining a block
+- `ByzantiumBlockReward`          = big.NewInt(3e+18) // Block reward in wei for successfully mining a block upward from Byzantium
+- `ConstantinopleBlockReward`     = big.NewInt(2e+18) // Block reward in wei for successfully mining a block upward from Constantinople
+
+!!! info
+
+    The block `miner` field is now populated with the rewarded account.
+
+### Istanbul
+
+Standard geth behaviour is that if a transaction has less gas than the accepted minimum
+(set using `--miner.gasprice`) then it is not mined by remote miner nodes.
+On Istanbul, it means the transaction is only processed when the current node becomes voted in as minter.
+
+The Istanbul block signer is rewarded for creating the block, using the account number derived from the nodekey of the miner.
+
+Note that `miner.SetEtherbase()`, and the command line flag `--miner.etherbase` have no effect for Istanbul.
+
+### Clique
+
+Standard geth behaviour is that if a transaction has less gas than the accepted minimum
+(set using `--miner.gasprice`) then it is not mined by remote miner nodes.
+On Clique, it means the transaction is only processed when the current node becomes voted in as minter.
+
+The Clique block signer is rewarded for creating the block through the coinbase account of the miner.
+The account is in the `extraData` field of the genesis block or has been added using `clique.propose()`
+Otherwise an "unathorized signer" error occurs.
```

Comment:
```suggestion
The account is in the `extraData` field of the genesis block or has been added using `clique.propose()`,
otherwise an `unauthorized signer` error occurs.
```

Reply:
Done